rakelib/lint.rake

```diff
@@ -0,0 +1,22 @@
+namespace :lint do
+  desc 'Install swiftlint'
+  task :install do |task|
+    swiftlint = `which swiftlint`
```

Comment:
No it was already failing before I even touched the rake file. I have a default ruby of 2.4 though, not 2.0 (or whatever the default is on macOS now)